docs/masks.rst

```diff
@@ -163,8 +163,11 @@ as well as a cutout with the data multiplied by the mask::
     >>> weighted_data = mask.multiply(hdu.data)
     >>> pf.close()
 
+Note that ``weighted_data`` will have zeros where the mask is zero; it therefore
+should not be used to compute statistics.  See `Masked Statistics` below.
+
 We can take a look at the results to make sure the source overlaps
-with the aperture::
+with the aperture
```

Comment:
You can keep a single ":" here.  The double "::" is what triggered the `doctest-skip::` showing up in the docs.